### ISSUE TITLE
`rusc` does not create config directory by default.

### ISSUE THREAD

**Issue by gBasil** [Drewol/kson-rs#33]
Running the code in the #32 pull request (though this is the same issue on the default branch) via `cargo run --bin rusc` causes the following terminal output:

```
     Running `target/debug/rusc`
2023-09-06T19:52:22.397Z WARN  [rusc] Could not find installed assets.
2023-09-06T19:52:22.397Z INFO  [rusc] Running anyway
2023-09-06T19:52:22.397Z INFO  [rusc::config] Loading game config from: "/Users/basil/Library/Application Support/usc/Main.cfg"
2023-09-06T19:52:22.397Z ERROR [rusc::config] No such file or directory (os error 2)
2023-09-06T19:52:22.399Z WARN  [rusc::config] No such file or directory (os error 2)
2023-09-06T19:52:22.604Z ERROR [rusc] Os { code: 2, kind: NotFound, message: "No such file or directory" }
2023-09-06T19:52:22.604Z INFO  [rusc] Skin Settings: []
thread 'main' panicked at 'No default font loaded', game/src/vg_ui.rs:179:48
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

This seemed to be the case on both macOS and Linux, though I only tried it on Linux once a while ago.

Even when the directory is made, you still have to add a Main.cfg with some values (I believe defining the skin, font folder, and song path) before the game successfully launches and adds the other values in for you.

**Comment by Drewol** [comment]
I pushed a patch a few minutes ago that *should* fix this ae6d903de43746b79e5a94ee4aba01c1148a4520

**Comment by Drewol** [comment]
with this commit, `cargo run --bin rusc` should be the preferred way
